extensions/amp-user-notification/0.1/amp-user-notification.css

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-user-notification {
+  visibility: hidden;
+}
+
+.amp-user-notification {
+  position: fixed;
```

Comment:
form our previous discussion if i remember correctly we are ok with users restyling amp-user-notification, but I was told users are never able to use `!important` so they can never override `position: fixed !important`